Description: Remove git ls-files usage from gemspec file
Author: Alexandre Ghiti <alexandre.ghiti@canonical.com>
Origin: other
Last-Update: 2022-01-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pygments.rb.gemspec
+++ b/pygments.rb.gemspec
@@ -28,5 +28,5 @@
   s.add_development_dependency 'rubocop', '~> 0.81.0'
   s.add_development_dependency 'test-unit', '~> 3.5.0'
 
-  s.files = `git ls-files -z`.split("\0").reject { |f| File.symlink?(f) }
+  s.files = (Dir.glob('**/*', File::FNM_DOTMATCH) - Dir.glob('debian/**/*', File::FNM_DOTMATCH)).reject { |f| File.directory?(f) }
 end
